lib/auth/tun.go

```diff
@@ -119,7 +119,11 @@ func (s *AuthTunnel) Start() error {
 }
 
 func (s *AuthTunnel) Close() error {
-	return s.sshServer.Close()
+	if s != nil && s.sshServer != nil {
```

Comment:
about these "ifs": see my comment in PR description. Calling `.Close()` too quickly results in all kinds of nasty things, here I minimized damange a bit.